Examples/Python/src/Vertexing.cpp

```diff
@@ -24,16 +24,25 @@ namespace py = pybind11;
 
 using namespace ActsExamples;
 using namespace Acts;
+using Seeder = ActsExamples::AdaptiveMultiVertexFinderAlgorithm::SeedFinder;
 
 namespace Acts::Python {
 
 void addVertexing(Context& ctx) {
   auto mex = ctx.get("examples");
+  auto& m = ctx.get("main");
+  auto seeder = m.def_submodule("seeder", "");
+
+  py::enum_<Seeder>(seeder, "SeedFinder")
+      .value("GaussianSeeder", Seeder::GaussianSeeder)
+      .value("AdaptiveGridSeeder", Seeder::AdaptiveGridSeeder)
+      .export_values();
```

Comment:
`Seeder` might be a bit too generic as we also have different seeders for track finding. Maybe `VertexSeeder` would be a better fit